src/miniapp.ts

```diff
@@ -0,0 +1,5 @@
+export declare namespace MiniApp {
```

Comment:
I think I had mentioned using `namespace`, but I think maybe it doesn't need to use it.

Reply:
This syntax is easier to read: https://www.typescriptlang.org/docs/handbook/namespaces.html#namespaced-validators
What do you think?

Reply:
I don't see much different 😃

Reply:
Generally, declarations are exported. So a usage along namespace looks 🤔 
https://www.typescriptlang.org/docs/handbook/modules.html#export

Reply:
I see. Let's me remove it if it's redundant.

Reply:
The export of function is still required because we use it as an interface function (using the implementation later from native sdk).